sass/_Quickview.scss

```diff
@@ -195,6 +195,10 @@
   }
 }
 
+.coveo-icon-for-quickview {
```

Comment:
This prevents the icon from being decentered from the parent if the cell's text is aligned to the right/left